lib/utils/defer.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Promisify a function of the form (value, options, cb)
+ * @param  {Function} asyncFunc The function
+ * @return {Function} 					The new function, as a Promise
```

Comment:
I followed JSDocs style in the original, but are you using JSDocs now to generate the docs?